tests/beaker_tests/cisco_vlan/vlan/standardvlan_provider_nondefaults.rb

```diff
@@ -99,10 +99,11 @@
       "resource cisco_vlan '128'", options)
     on(agent, cmd_str) do
       search_pattern_in_output(stdout,
-                               { 'ensure'    => 'present',
-                                 'shutdown'  => 'true',
-                                 'state'     => 'suspend',
-                                 'vlan_name' => 'DESCR-VLAN0128' },
+                               { 'ensure'     => 'present',
+                                 'mapped_vni' => '128000',
```

Comment:
I made the changes in beaker scripts and tested them with a hack in utilitylib.rb which returns n9k and n7k in separate cases. Somehow utilitylib 'platform' method is not detecting my agent as n9k. Here is my hardware:

Hardware
  cisco NX-OSv Chassis 
  Intel(R) Xeon(R) CPU E5-2697 v2 @ 2.70GHz with 8165880 kB of memory.

  Device name: agent-lab1-nx
  bootflash:    1307446 kB
Kernel uptime is 11 day(s), 23 hour(s), 0 minute(s), 24 second(s)

Regardless, chasing this issue in utilitylib is not in scope of this CR. So I tested the implementation with a hack.